### PR TITLE
Move buy book button from toolbar to bottom of the page

### PR DESCRIPTION
**Pull request by KarinaMendez2**
for: https://app.zenhub.com/workspaces/openstax-unified-5b71aabe3815ff014b102258/issues/openstax/unified/976

### CODE REVIEW

File: src/app/content/components/BuyBook.tsx [hunk 1]
```diff
@@ -1,26 +1,26 @@
 import React from 'react';
 import { FormattedMessage } from 'react-intl';
 import { useSelector } from 'react-redux';
-import BuyBookIcon from '../../../../assets/buy-book-icon.svg';
-import { buyBookLink } from '../../selectors';
-import * as Styled from './styled';
+import { buyBookLink } from '../selectors';
+import * as Styled from './Toolbar/styled';
```
Comment (PiotrKozlowski) [inline src/app/content/components/BuyBook.tsx:5]: I guess this component shouldn't import styles from unrelated location

File: src/app/content/components/Toolbar/styled.tsx [hunk 2]
```diff
@@ -319,13 +330,21 @@ export const SidebarControl = styled(OpenSidebarControl)`
 
 // tslint:disable-next-line:variable-name
 export const BuyBookWrapper = styled.a`
```
Comment (PiotrKozlowski) [inline src/app/content/components/Toolbar/styled.tsx:332]: margin top and bottom differs a little bit from the design

**Review by PiotrKozlowski — CHANGES_REQUESTED** [review]

**Review by PiotrKozlowski — APPROVED** [review]

File: src/app/content/components/BuyBook.tsx [hunk 3]
```diff
@@ -33,6 +37,7 @@ const BuyBookText = styled.span`
   margin: 0;
   font-weight: 600;
   line-height: 1.9rem;
+  overflow: hidden;
```
Comment (PiotrKozlowski) [inline src/app/content/components/BuyBook.tsx:40]: This seems to be unnecessary

Comment (KarinaMendez2) [reply src/app/content/components/BuyBook.tsx:40]: It isn’t, prevents the span from being tabable on firefox.

Comment (PiotrKozlowski) [reply src/app/content/components/BuyBook.tsx:40]: Is this span need for something then?

Comment (TomWoodward) [reply src/app/content/components/BuyBook.tsx]: i tried removing it and it looked the same, so i'm pushing that, karina can correct me later if it was needed for something

**Review by TomWoodward — APPROVED** [review]

**Pull request merged**